docs/index.md

```diff
@@ -97,12 +98,13 @@ There you will see a list of the messages you uploaded. By using the box
 on the right, you can filter the list to show only the messages that are
 enabled.
 
-If you click on a message, it will show you its details. Here you can 
-edit it, and disable or enable it.
+If you click on a message, it will show you its details.
 
-If you disable a message, mPACT will not send it.
+**NOTE:** Messages are scheduled to be sent, as they are, when schedules
+are uploaded. Changes made to messages in the admin interface will not
+affect messages that are already scheduled to be sent.
 
-You can also use the admin interface to delete scheduled messages and
-create new ones. If you download schedules from the normal interface
-again, you will see that the changes you made in the admin interface are
-present in the downloaded spreadsheet.
+If you download schedules from the normal interface, you will see that
+any changes you made in the admin interface are present in the
+downloaded spreadsheet. By uploading the spreadsheet, mPACT will
+re-schedule the messages, and bring those changes into effect.
```

Comment:
should we just fix this to automatically run any time a scheduled message changes? feels a little silly

Reply:
agreed, yep

Reply:
https://github.com/dimagi/mpact/pull/49/